pkg/api/status.go

```diff
@@ -25,6 +25,7 @@ const (
 	ErrStatusValidatingMerkleRoots  StatusCode = 469
 	ErrStatusFrozenPolicy           StatusCode = 471
 	ErrStatusFrozenConsensus        StatusCode = 472
+	ErrStatusCumulativeFees         StatusCode = 473
```

Comment:
This needs to be added also here I think: `doc/errors.md`

Reply:
Maybe we need here a comment like: "if a new error is added, don't forget to add to 'doc/errors.md'"

Reply:
I thought I add this to doc/ in follow up PR